.github/workflows/publish.yml

```diff
@@ -51,20 +51,14 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-
-      - name: Login to Docker Hub
-        uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}       
    
       - name: package and release charts
         env:
           HELM_GCS_CREDENTIALS: ${{ secrets.HELM_GCS_CREDENTIALS }}
           HELM_GCS_REPOSITORY: ${{ secrets.HELM_GCS_REPOSITORY }}
             # GITHUB_REF will be refs/tags/docker-MAJOR.MINOR.PATCH
         run: |
-            CHART_VERSION=$(git describe --abbrev=0)
+            CHART_VERSION=$(echo ${GITHUB_REF} | cut -d/ -f 3)
```

Comment:
Does the tag ref not come down here? In other words, the old command seemed simpler unless there's a reason to change

Reply:
I am not sure about this one. Let's use the earlier one and if any issue is there we can switch. I found most of the workflows using GITHUB_REF way